src/quicktype-core/language/ruby/index.ts

```diff
@@ -159,12 +160,13 @@ export class RubyRenderer extends ConvenienceRenderer {
 
     private dryType(t: Type, isOptional: boolean = false): Sourcelike {
         const optional = isOptional ? ".optional" : "";
+        const intTypeDeclaration = this._options.legacyGemVesion ? "Int" : "Integer";
```

Comment:
I'm actually thinking if we should even use `Types::Int` instead of `Types::Integer` with legacy versions. It was deprecated since May 2018, the 0.13 release (see [changelog](https://github.com/dry-rb/dry-types/blob/master/CHANGELOG.md#0130-2018-05-03))

If we decide to support `Types::Int`, we'll have to add `require 'dry/types/compat/int'` though – it wouldn't work otherwise.

What do you think?

Reply:
If only need to add one require to keep support, I think it can be done 🤔